database/dbSchoolSuppliesForm.php

```diff
@@ -43,5 +43,39 @@ function isBackToSchoolFormComplete($childID) {
     return $complete;
 }
 
+
+function getSchoolSuppliesSubmissions() {
+    $conn = connect();
+    $query = "SELECT * FROM dbSchoolSuppliesForm JOIN dbChildren USING(id);";
+    $result = mysqli_query($conn, $query);
+    if(mysqli_num_rows($result) > 0){
+        $submissions = mysqli_fetch_all($result, MYSQLI_ASSOC);
+        mysqli_close($conn);
+        return $submissions;
+    }
+    mysqli_close($conn);
+    return [];
+}
+
+function getSchoolSuppliesSubmissionsFromFamily($familyId) {
+    require_once("dbChildren.php");
+    $children = retrieve_children_by_family_id($familyId);
+    if (!$children){
+        return [];
+    }
+    $childrenIds = array_map(function($child) {
+        return $child->getId();
+    }, $children);
+    $joinedIds = join(",",$childrenIds);
+    $conn = connect();
+    $query = "SELECT * FROM dbSchoolSuppliesForm JOIN dbChildren ON dbSchoolSuppliesForm.child_id = dbChildren.id WHERE dbSchoolSuppliesForm.child_id IN ($joinedIds)";
```

Comment:
When I go in to view results for school supplies form, I get all the correct info from the school supplies database, but I also see children info taken from dbChildren. I think the issue may come from this SELECT statement

Reply:
Yeah thats intentional for now, since otherwise they would only be able to see the associated child id and I dont know if thats useful to them